components/SectionTextBackground/SectionTextBackground.tsx

```diff
@@ -0,0 +1,67 @@
+import { Variant } from "../Variant/Variant";
+import styled from "styled-components";
+
+const StyledSectionTextBackground = styled.section<Variant>`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: ${({ variant }) =>
+    variant === "dark" ? "var(--me-color-gray-500)" : "var(--me-color-white)"};
+  color: ${({ variant }) =>
+    variant === "dark" ? "var(--me-text-inverted)" : "var(--me-text-gray-700)"};
+`;
+
+const StlyedHeadlineGroup = styled.div`
+  justify-content: flex-end;
+  align-self: flex-end;
+  padding-bottom: 5rem;
+  margin: 0.5rem;
+`;
+
+const StyledParagraph = styled.div`
+  justify-content: flex-end;
+  align-self: flex-end;
+  padding-left: 16rem;
+  padding-bottom: 5rem;
+  margin-right: 5rem;
+`;
+
+interface ContentPropsTwo extends Variant {
```

Comment:
What does two mean? If it's a counter you could find a more appropriate name. Why not SectionTextBackgroundProps?